pkg/apis/management.cattle.io/v3/machine_types.go

```diff
@@ -300,34 +322,45 @@ var (
 	NodeDriverConditionInactive   condition.Cond = "Inactive"
 )
 
-type Condition struct {
-	// Type of cluster condition.
-	Type string `json:"type"`
-	// Status of the condition, one of True, False, Unknown.
-	Status v1.ConditionStatus `json:"status"`
-	// The last time this condition was updated.
-	LastUpdateTime string `json:"lastUpdateTime,omitempty"`
-	// Last time the condition transitioned from one status to another.
-	LastTransitionTime string `json:"lastTransitionTime,omitempty"`
-	// The reason for the condition's last transition.
-	Reason string `json:"reason,omitempty"`
-	// Human-readable message indicating details about last transition
-	Message string `json:"message,omitempty"`
-}
-
 type NodeDriverSpec struct {
+	// DisplayName specifies the publicly visible name of the driver shown in the Rancher UI.
+	// +optional
 	DisplayName string `json:"displayName"`
```

Comment:
Can you add `// + kubebuilder:validation:MaxLength=57` to this? We currently use it to generate the `<driver display name>config` objects, so anything above 57 would cause the driver to be unusable for provisioning v2, and since RKE1 prov is going away it's the only case we need to worry about.